speech-to-speech/SpeechToSpeech/README.md

```diff
@@ -0,0 +1,88 @@
+# Cloud Speech to Speech Translation gRPC Swift Sample
+
+This app demonstrates how to make gRPC connections to the(API Integration):
+1. Make streaming gRPC connections to the [Cloud Speech API](https://cloud.google.com/speech/) to the recognize speech audio. Receiving Transcript.
+2. Send Transcript to [Translate API](https://cloud.google.com/translate/)
+3. Send Translated text to [Text-to-Speech API](https://cloud.google.com/text-to-speech/)
+4. [Get list of voices](https://cloud.google.com/text-to-speech/docs/list-voices)
+
+
+To call the APIs from iOS, you need to provide authorization tokens with each request for authentication. To get this token, this sample uses a Firebase Function (in Node.js) to generate these tokens on the behalf of a service account.
+
+## Prerequisites
+- An OSX machine or emulator
+- [Xcode 8 beta 6][xcode] or later
+- [Cocoapods][cocoapods] version 1.0 or later
+
+## Setup
+
+- Create a project (or use an existing one) in the [Google Cloud Console][cloud-console]
+- [Enable billing][billing] and the
+    - [Speech API](https://console.cloud.google.com/apis/library/speech.googleapis.com).
+    - [Translate API](https://console.cloud.google.com/apis/library/translate.googleapis.com).
+    - [Text-to-Speech API](https://console.cloud.google.com/apis/library/texttospeech.googleapis.com).
+    - [IAM Service Account Credentials API](https://console.cloud.google.com/apis/library/iamcredentials.googleapis.com).
+- [Create a Service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) with the following IAM role: `Cloud Translation API User`. Example name: `translate-client`. ([For more info on: how to add roles to a Service Account](https://cloud.google.com/iam/docs/granting-roles-to-service-accounts#granting_access_to_a_service_account_for_a_resource))
+
+###  Setup the app
+- Clone this repository `git clone https://github.com/GoogleCloudPlatform/ios-docs-samples.git` 
+- `cd ios-docs-samples/dialogflow/speech-to-speech/` 
```

Comment:
No `dialogflow` 
Should be updated to the new folder layout.

Reply:
Updated.